doc/ideas/gameplay.md

```diff
@@ -152,16 +152,73 @@ Multiplayer
+   - admin changes still possible for special use cases
+  - when starting the tournament every player will download the important information from their pre-chosen and fixed parts and everything will be pre-configured and grey (if no spontaneous changes are allowed)
+
+  - this brings the possibility to bring all the systems taking part in the tournament to one comparable and static setup (no admin RE because of forgotten MQ anymore)
+
+  - everything regarding the tournament (replays, civdrafts, etc.) should be stored in this container after the event (so you could easily import it into your OpenAge and replay games -> also see 2. Spectating/Casting-Mode)
+
+  - as an organiser of a tournament you should have the possibility to manage the whole tournament inside the OpenAge platform (not: making promo there, drafting here, hosting matches there, and so on)
+
+  - also the possibility to directly invite a player from your OpenAge-account based on characteristics (like direct-invites for 1v1-Top15 Ladder-Tournament or Teamgames)
+   - teams can connect their accounts to real clans reaching together a team elo depending on the combination of actual players (TatoH + DauT + TheViper > TatoH + DauT + Slam)
+
+  - make it easier to qualify for your tournament (e.g. as a potential player -> Open "Tournaments" in multiplayer menu -> open "Qualifiers" -> Play games with fixed tournament settings -> maybe qualify for Tournament)
+
+  - After clicking on the Tournament-Menu-Item you could have an overview of upcoming and open tournaments, with a calendar view of the next tournaments, a timeplan, the brackets and maybe even the possibility to one-click-spectate a Live-Tournament if you want (see 2. Spectating/Casting-Mode)
+
+  - depending on the architecture of the multiplayer it could be even possible to spare out all that client modeling and just integrate an basic in-game browser which connects just to an OpenAge-community-server and from there you could manage everything. To play a game the user could just click on a link from the game browser which gives data to an internal protocol like openage://<game-id>/<foo>/<bar>/<link-to-settings-file>/<...>
+
+   - this would make the client thinner
+   - but: taking too much multiplayer features out of the client leaves it helpless for private matches -> thin grade
+
+
+### Spectating/Casting
+
+ - the casting mode could be an advanced spectating mode, with more features than just replaying/live-spectating the game
+
+ - ZOOM! (also in-game not just for casters)
+ - maybe even turning the game in 90° angles
+  - the old models could stay 2D and just be fit into the turned environment(?)
+
+ - the feature set of CaptureAge could be understood as a spectator mode
+  - if they add PIP and replay-functions goes more into direction what's understood in this context as casting mode
```

Comment:
what is PIP?

Reply:
Picture-in-Picture, i'll add the explanation to the file